docs/requirements.txt

```
@@ -1,5 +1,5 @@
 aenum
-apache-airflow==2.10.2
+apache-airflow==2.10
```

```suggestion
apache-airflow>=2.10.3,<2.11
```
could this be better if the vulnerability was fixed in 2.10.3?